frontend/nginx/nginx.conf

```diff
@@ -0,0 +1,17 @@
+server {
+
+  listen 80;
+
+  location / {
+    root   /usr/share/nginx/html;
+    index  index.html index.htm;
+    try_files $uri $uri/ /index.html;
+  }
+
+  error_page   500 502 503 504  /50x.html;
+
+  location = /50x.html {
+    root   /usr/share/nginx/html;
+  }
+
+}
```

Comment:
Good call on nginx here, maybe we should discuss for another one in front of frontend + backend so as to secure both AND the file server
It'll be also useful to handle SSL/HTTPS on the whole app level